drivers/i2c/i2c_ll_stm32.c

```diff
@@ -289,9 +289,23 @@ static void i2c_stm32_irq_config_func_##name(struct device *dev)	\
 
 #endif /* CONFIG_I2C_STM32_INTERRUPT */
 
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_i2c_v2)
+#define DEFINE_TIMINGS(name)						\
+	static const uint32_t i2c_timings_##name[] =			\
+		DT_PROP_OR(DT_NODELABEL(name), timings, {});
+#define USE_TIMINGS(name)						\
+	.timings = (const struct i2c_config_timing *) i2c_timings_##name, \
+	.n_timings = ARRAY_SIZE(i2c_timings_##name),
+#else /* V2 */
```

Comment:
#else case is for `V1`?

Reply:
yes - this timing option is not applicable for v1